pkg/api/controller.go

```diff
@@ -27,11 +27,13 @@ type Controller struct {
 	Router     *mux.Router
 	ImageStore *storage.ImageStore
 	Log        log.Logger
+	Audit      log.Logger
 	Server     *http.Server
 }
 
 func NewController(config *Config) *Controller {
-	return &Controller{Config: config, Log: log.NewLogger(config.Log.Level, config.Log.Output)}
+	var Log, Audit = log.NewLogger(config.Log.Level, config.Log.Output, config.Log.Audit)
```

Comment:
log,audit := log.NewLogger(config.Log.Level, config.Log.Output, config.Log.Audit) instead of 
var Log, Audit = log.NewLogger(config.Log.Level, config.Log.Output, config.Log.Audit)